src/index.ts

```diff
@@ -1 +1,32 @@
-//this is the main file which has the final functions all compiled in one place
\ No newline at end of file
+import TelegramBot from 'node-telegram-bot-api';
+import {Builder, By, until} from 'selenium-webdriver';
+import fs from 'fs';
+import strings from '../strings';
+import { setDefaultValues, mountDBObject } from './utils/dbutils';
+import low from 'lowdb';
+import  { adapter } from './utils/dbutils';
+//TODO: find out why I can't import dotenv
+/* Load dotenv before code to apply environment variables */
+require('dotenv').config()
+//TODO: find out why I can't import AsciiTable
+const AsciiTable = require('ascii-table');
+
+/* Secret, Personal Telegram Token given by botfather */
+const token = process.env.BOT_TOKEN;
```

Comment:
Use type: string

Reply:
I'll have to user string | undefined for now, but I'll think of another way to deal with this later.